packages/lesswrong/components/posts/PostsByVote.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useMulti } from '../../lib/crud/withMulti';
+import { registerComponent, Components } from '../../lib/vulcan-lib';
+
+const PostsByVote = ({postIds, year}:{postIds: Array<string>, year: number}) => {
+    const { PostsItem2, ErrorBoundary } = Components
+
+    const before = `${year + 1}-01-01`
+    const after = `${year}-01-01`
+
+    const { results: posts } = useMulti({
+      terms: {
+        view: "nominatablePostsByVote",
+        postIds: postIds,
+        before: before,
+        after: after
+      },
+      collectionName: "Posts",
+      fragmentName: "PostsList",
+      limit: 1000,
```

Comment:
I'd be very tempted to put a lower limit here and load more if needed, but I'll leave that to your discretion.